tests/foreman/api/test_computeprofile.py

```diff
@@ -34,6 +34,8 @@ def test_positive_create_with_name(self):
         :expectedresults: Compute Profile is created
 
         :CaseImportance: Critical
+
+        :CaseLevel: Component
```

Comment:
Can you please define the `CaseLevel` at module level so that you no need to set this for every test where it works ?

Reply:
@vijay8451 It's perfectly fine to define the same metadata for each case individually, too. Hope you don't consider this comment blocking, because I think it is not. It's mostly personal preference thing.

Reply:
@mirzal you are right ... it is a suggestion only not blocker ...
However when I checked the test file I see that at module level it is still requires to change too: https://github.com/SatelliteQE/robottelo/blob/bf73f96589b80a30375f5dd41b680eca5c084bc5/tests/foreman/api/test_computeprofile.py#L10

Hence above may help to achieve both part of it.